prdoc/pr_5891.prdoc

```diff
@@ -0,0 +1,33 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Add benchmark overhead command to frame-omni-bencher
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      This adds the benchmark overhead command to the `frame-omni-bencher` library. This allows
+      para- and relay chain teams to generate extrinsic and block base weights.
+
+crates:
+  - name: sc-chain-spec
+    bump: none
+  - name: polkadot-service
+    bump: major
+  - name: frame-benchmarking-cli
+    bump: major
+  - name: cumulus-client-parachain-inherent
+    bump: patch
+  - name: polkadot-cli
+    bump: none
+  - name: polkadot-omni-node-lib
+    bump: none
+  - name: polkadot-omni-node
+    bump: none
+  - name: polkadot-parachain-bin
+    bump: none
```

Comment:
None is mostly for the case that only doc stuff was changed - no real code. Everything else is at least patch.